src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -245,6 +246,30 @@ xclbinutil --add-section PDI:RAW:$APU_PDI --output $FW_FILE
 if [[ ! -e $FW_FILE ]]; then
 	error "failed to generate XSABIN"
 fi
+
+# Generate PS Kernel xclbin
+# Hardcoding the ps kernel xclbin name to ps_kernels.xclbin
+# We can create one xclbin per PS Kernel also based on future requirements
+PS_KERNELS_XCLBIN="$BUILD_DIR/lib/firmware/xilinx/ps_kernels/ps_kernels.xclbin"
+#Extract ps_kernels_lib from rootfs tar
+tar -C $BUILD_DIR -xf $IMAGES_DIR/rootfs.tar.gz ./usr/lib/ps_kernels_lib
+if [ $? -eq 0 ]; then
+    PS_KERNEL_DIR=$BUILD_DIR/usr/lib/ps_kernels_lib
+    ps_kernel_command="xclbinutil --output $PS_KERNELS_XCLBIN "
```

Comment:
This is an xclbinutil hang bug that occurs if objdump returns more then 4K of data.
https://github.com/Xilinx/XRT/pull/6203

This only occurs on platforms that are using boost version 16.04 and greater.  Since, you haven't encountered this issue.  I am assuming that all is working for you.

Reply:
i didn't face any issue while building simple PS Kernel.  Jason is anyway going to add his PS Kernels once he is ready.